contracts/IdleTokenFungible.sol

```diff
@@ -544,16 +534,25 @@ contract IdleTokenFungible is Initializable, ERC20Upgradeable, ReentrancyGuardUp
    * Calculate gain and save eventual fees in unclaimedFees
    */
   function _updateFeeInfo() internal {
+    // remove fees
+    uint256 _currNAV = _getCurrentPoolValue() - unclaimedFees;
```

Comment:
I'm a bit confused. I want to check my understanding. 

`_getCurrentPoolValue ` includes fees already accounted  and fees that are yet to be accounted.
`_currNAV` is NAV without fees already accounted
`lastNAV` is NAV without that are yet to be accounted. we update `unclaimedFees` and subtract ` lastNAV` by fees to be accounted.

Reply:
before claiming fees, we always update fees info by `_updateFeeInfo`.
right after the `_updateFeeInfo` ,  fees are yet to be accounted is zero.

Reply:
`_getCurrentPoolValue` is the full value of the pool, everything included yes
`_currNAV` yes correct
`lastNAV` which is the only value saved, is with all fees removed (the one already accounted in `unclaimedFees` + the newly calculated fees that are yet to be accounted but are accrued since the last update).

`_updateFeeInfo` is called before every interaction mint/redeem/rebalance and yes after is it called `_calculateFees` will return 0 basically because `_currNAV` will be equal to `lastNAV` so your understanding is correct on the flow

Reply:
ok